versioned_docs/version-22.10/monitoring/basic-objects/hosts.md

```diff
@@ -6,127 +6,137 @@ title: Creating hosts manually
+* **Event Handler**: command to be executed if the event handler is enabled.
+* **Args**: arguments of the events handler command.
 
 ## Host Extended Infos tab
 
 ### Monitoring engine
 
-* The **Note URL** field defined a URL that can be used to give more information on the host.
-* The **Note** field permits us to add optional notes concerning the host.
-* The **Action URL** field defined a URL normally use for giving information on actions on the host (maintenance, etc.).
-* The **Icon** field indicates the icon use for the host.
-* The **Alt Icon** field is the text use if the icon cannot be Display.
-* The **Host severity** field indicates the severity level of the host.
-* The **Status Map Image** field defined the logo for Centreon Map module.
-* The **Geographic coordinates** field defines geographical coordinates used by the Centreon MAP module to position the resource on a map.
-  Define "Latitude,Longitude", for example for Paris coordinates set "48.51,2.20"
-* **Enable/disable resource**: This setting determines whether the host and its services must be monitored or not. If the host is disabled, it doesn't appear on the **Resources Status** page.
+* **Note URL**: clickable URL that will appear in the **Notes** column on the **Resource Status** page (the **Notes** column must be displayed for this).
+* **Note**: optional information concerning the host visible in the Resource Status page (the **Notes** column must be displayed for this).
+* **Action URL**: URL normally used for giving information on actions on the host (maintenance, etc.).
+* **Icon**: icon used for the host, it can be seen in a variety of places such as the ressource status page. A 40x40 pixels format is recommended.
+* **Alt Icon**: optional string shown when the **Icon** cannot be displayed.
+* **Host severity**: [severity](../categories.md#severities) level of the host. This is a special type of category that also allows you to sort the Resource Status by severity level.
+* **Status MAP Image**: logo for the [Centreon MAP module](../../graph-views/introduction-map.md).
+* **Geographic coordinates**: geographical coordinates used by the Centreon MAP module to position the resource on a map.
+  Define "Latitude,Longitude"; for example, the set of coordinates for Paris is "48.51,2.20"
 
-The fields presented below are obsolete:
+The **2d Coords** and **3d Coords** fields are obsolete and have no impact on the host.
 
-* **2d Coords**
-* **3d Coords**
-
-### Access groups
+### Additional Information
 
-* The **ACL Resource Groups** (only displayed for non administrator) allows to link this host to an hostgroup in order
-  to visualize it (see *[Access Control Lists](../../administration/access-control-lists.md)* chapter).
+* **Enable/disable resource**: determine whether or not the host and its services must be monitored. If the host is disabled, it does not appear on the **Resources Status** page.
```

Comment:
```suggestion
* **Enable/disable resource**: determine whether or not the host and its services must be monitored. If the host is disabled, it will still show on the list of hosts but will not appear on the **Resources Status** page. Warning, disabled hosts still count towards the number of hosts included with your licence.
```